mkdocs/docs/HPC/alphafold.md

```diff
@@ -0,0 +1,232 @@
+{% set directory="/arcanine/scratch/gent/apps/AlphaFold" %}
+{% set version="20230310" %}
+{% set readme="https://github.com/deepmind/alphafold/blob/main/README.md" %}
+## What is AlphaFold?
+
+AlphaFold is an AI system developed by DeepMind that predicts a protein’s 3D structure from its amino acid sequence. 
+It aims to achieve accuracy competitive with experimental methods.
+
+AlphaFold DB provides open access to over 200 million protein.
```

Comment:
I would remove this line, we don't need to "sell" AlphaFold here, just give some context.